doc/UserGuide.md

```diff
@@ -43,9 +43,9 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > be seen using the `viewall` command.
 > 
 > Persons can have any number of tags (including 0)
-
+> Format of the address : [block number][street name][unit number][postal code]
 Examples: 
```

Comment:
No need to add a newline here, instead change the format above, line 37, `Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` `.